crates/nu-command/tests/commands/ls.rs

```diff
@@ -399,9 +406,12 @@ fn fails_with_ls_to_dir_without_permission() {
         ));
 
         assert!(
-            actual
+            (actual_with_path_arg
                 .err
                 .contains("The permissions of 0 do not allow access for this user")
+                && actual_in_cwd
```

Comment:
Just one thing, I think you can make a separate `assert!`.  It's easier to know which assertion failed.